src/More/ENDebugMenu.tsx

```diff
@@ -98,16 +98,16 @@ const ENDebugMenu = ({ navigation }: ENDebugMenuProps): JSX.Element => {
         <ScrollView>
           <View style={style.section}>
             <DebugMenuListItem
-              label="Show Last Processed File Path"
+              label="Simulate Exposure"
```

Comment:
Should these labels be translated? It seems like this is intended to be used by clients directly so it might need to be a higher level of polish on the functionality.